mesos-rxjava-client/src/main/java/com/mesosphere/mesos/rx/java/MesosClientBuilder.java

```diff
@@ -163,6 +166,108 @@ private MesosClientBuilder() {}
     }
 
     /**
+     * Instructs the HTTP byte[] stream to be composed with reactive pull backpressure such that
+     * a burst of incoming Mesos messages is handled by an unbounded buffer rather than a
+     * MissingBackpressureException.
+     *
+     * As an example, this may be necessary for Mesos schedulers that launch large numbers
+     * of tasks at a time and then request reconciliation.
+     *
+     * @return this builder (allowing for further chained calls)
+     * @see <a href="http://reactivex.io/documentation/operators/backpressure.html">ReactiveX operators documentation: backpressure operators</a>
+     */
+    @NotNull
+    public MesosClientBuilder<Send, Receive> onBackpressureBuffer(
+
+    ) {
+        this.byteStreamTransformer = observable -> observable.onBackpressureBuffer();
+        return this;
+    }
+
+
+    /**
+     * Instructs the HTTP byte[] stream to be composed with reactive pull backpressure such that
+     * a burst of incoming Mesos messages is handled by a bounded buffer rather than a
+     * MissingBackpressureException. If the buffer is overflown, a {@link java.nio.BufferOverflowException}
+     * is thrown.
+     *
+     * As an example, this may be necessary for Mesos schedulers that launch large numbers
+     * of tasks at a time and then request reconciliation.
+     *
+     * @param capacity number of slots available in the buffer.
+     * @return this builder (allowing for further chained calls)
+     * @see <a href="http://reactivex.io/documentation/operators/backpressure.html">ReactiveX operators documentation: backpressure operators</a>
+     */
+    @NotNull
+    public MesosClientBuilder<Send, Receive> onBackpressureBuffer(
+        @NotNull final long capacity
+    ) {
+        this.byteStreamTransformer = observable -> observable.onBackpressureBuffer(capacity);
+        return this;
+    }
+
+    /**
+     * Instructs the HTTP byte[] stream to be composed with reactive pull backpressure such that
+     * a burst of incoming Mesos messages is handled by a bounded buffer rather than a
+     * MissingBackpressureException. If the buffer is overflown, your own custom onOverflow callback
+     * will be invoked, and the overflow will mitigate the issue based on the {@link BackpressureOverflow.Strategy}
+     * that you select.
+     *
+     * <ul>
+     *     <li>{@code BackpressureOverflow.Strategy.ON_OVERFLOW_ERROR} (default) will {@code onError} dropping all undelivered items,
```

Comment:
Here, when referring to the enum values instead of using `{@code ...}` we can use `{@link ...}` and the javadoc will actually link the properties and javadocs together for use. (There is one syntactic thing that has to change however, when referring to the enum value we use `#` instead of `.`)

So instead of `{@code BackpressureOverflow.Strategy.ON_OVERFLOW_ERROR}` try `{@link BackpressureOverflow.Strategy#ON_OVERFLOW_ERROR}`